yarn.lock

```diff
@@ -975,19 +975,11 @@ append-transform@^1.0.0:
   dependencies:
     default-require-extensions "^2.0.0"
 
-aproba@^1.0.3, aproba@^1.1.1:
+aproba@^1.1.1:
```

Comment:
This file is not supposed to be changed. Please revert back the file to the original state.